Worker/worker.hocon

```diff
@@ -18,7 +18,7 @@ akka {
             "akka.tcp://MyCluster@localhost:4053"]
         roles = ["worker"]
 
-        downing-provider-class = "Akka.Cluster.SplitBrainResolver.SplitBrainResolverDowningProvider, Akka.Cluster.SplitBrainResolver"
+        downing-provider-class = "Akka.Cluster.SplitBrainResolver, Akka.Cluster"
```

Comment:
Need this to make sure the `SplitBrainResolver` is active on all nodes - no need to refer to that external package. It's built directly into Akka.Cluster.

When this is correctly configured, the node will be kicked out of the cluster shortly after being detected as unreachable and split from the referee:

```
[16:34:16 INF] Adding number 3 and number 8
[16:34:18 INF] Adding number 3 and number 2
[16:34:18 INF] A network partition detected - unreachable nodes: [akka.tcp://MyCluster@localhost:53984], remaining: [akka.tcp://MyCluster@localhost:4053, akka.tcp://MyCluster@localhost:54017]
[16:34:18 INF] A network partition has been detected. KeepReferee(refereeAddress: akka.tcp://MyCluster@localhost:4053, downIfLessThanNodes: 1) decided to down following nodes: [Member(address = akka.tcp://MyCluster@localhost:53984, Uid=1786851500 status = Up, role=[worker], upNumber=3)]
[16:34:18 INF] Cluster Node [akka.tcp://MyCluster@localhost:54017] - Marking unreachable node [akka.tcp://MyCluster@localhost:53984] as [Down]
[16:34:19 WRN] Association to [akka.tcp://MyCluster@localhost:53984] having UID [1786851500] is irrecoverably failed. UID is now quarantined and all messages to this UID will be delivered to dead letters. Remote actorsystem must be restarted to recover from this situation.
[16:34:20 INF] Adding number 0 and number 1
[16:34:22 INF] Adding number 4 and number 3
```